### DISCUSSION TITLE
This plugins does too much

### DISCUSSION THREAD

**Discussion by TobiaszCudnik** [Vinzent03/obsidian-git#466] [General]
This plugins does too much (IMO).

Instead of providing a reasonable replacement for the paid sync with a git backend, it tries to reimplement a git client in a markdown editor.

I have to stage changes? "Stage all" isnt even available as a command. The defaults arent sane, you have to init the repo manually, look for a sidebar, read through a convoluted setting page which magically changes at one point.

Kudos to the author for the work, but someone has missed the point. If one needs a git client, he can install a git client. 99% of people want a cross-platform sync, nothing more.

Just my opinion.

**Comment by Vinzent03** [comment]
The sidebar is completely optional. You can just use the "backup" command, which stages everything and commits everything and pushes that. Depending on setting is also pulls before.
